src/SFA.DAS.FAT.Web/Startup.cs

```diff
@@ -82,7 +83,8 @@ public void ConfigureServices(IServiceCollection services)
 
         services.AddServiceRegistration(_configuration);
         services.AddMediatR(cfg => cfg.RegisterServicesFromAssemblies(typeof(GetCourseQueryHandler).Assembly));
-        services.AddMediatRValidation();
+
+        services.AddValidatorsFromAssembly(typeof(GetCourseQueryValidator).Assembly);
```

Comment:
Do we not also need to add the GetCourseProviderDetailsQueryValidator here?

Reply:
no you only need to do this for one and one only validator, have a look at PR web (either one) to see

Reply:
Yep, fair enough